cpp/ql/lib/semmle/code/cpp/ir/implementation/raw/internal/TranslatedStmt.qll

```diff
@@ -777,6 +872,12 @@ class TranslatedIfStmt extends TranslatedStmt, ConditionContext {
 abstract class TranslatedLoop extends TranslatedStmt, ConditionContext {
   override Loop stmt;
 
+  override Instruction getALastInstructionInternal() {
+    result = this.getCondition().getALastInstruction() // FIXME: how do we handle the branch here
```

Comment:
Also this one

Reply:
Good point - I handled for loops but it turns out while loops allow a declaration as well.